packages/cozy-client/src/cli/index.js

```diff
@@ -63,7 +63,12 @@ const mkServerFlowCallback = serverOptions => authenticationURL =>
           'debug',
           'OAuth callback server started, waiting for authentication'
         )
-        opn(authenticationURL, { wait: false })
+        if (serverOptions.shouldOpenAuthenticationPage !== false) {
+          opn(authenticationURL, { wait: false })
+        }
+        if (serverOptions.onListen) {
+          serverOptions.onListen({ authenticationURL })
+        }
```

Comment:
There is no documentation about `serverOptions` ?

Reply:
Fixed in db5eab4